### ISSUE TITLE
X-405-H - adjustments and upgrades

### ISSUE THREAD

**Issue by blowfishpro** [KSP-RO/RealismOverhaul#1819]
Been doing a bit of research on the canceled X-405-H engine intended for the Vega upper stage of Atlas

Sources:
1) [Space Launch Report](http://www.spacelaunchreport.com/atlasnot.html) (Ed Kyle's site, I believe we've considered this to be reliable in the past)
2) [Alternate Wars](http://www.alternatewars.com/BBOW/Space_Engines/GE_Engines.htm) (Big Book of Warfare / Space Engine Encyclopedia)
3) [_History of Liquid Propellant Rocket Engines_](https://books.google.com/books?id=s1C9Oo2I4VYC&q=vega#v=snippet&q=vega&f=false) (book by George Paul Sutton)

Current stats in RO:
* Thrust: 156.3 kN
* Isp: 311s
* Ignitions: 2
* Burn time: 195s

1) Stats from Space Launch Report
* Engine Name: 405H-2
* Thrust: 156.3 kN
* Isp: 311s

2) Stats from Alternate Wars
* Engine Name: 405H
* Thrust: 150.5 kN
* Isp: 300s
* Expansion ratio: 25
* Burn time/ignitions: design conditions not specified, but was ground tested at 2000s and 12 ignitions

3) Stats from _History of Liquid Propellant Rocket Engines_
* Engine Name: 405H
* Thrust: 150.5 kN
* Isp: 300s
* Expansion ratio: 25
* Ignitions: 3 (using hypergolic ignition fluid)

There are definitely two sets of thrust/Isp numbers.  So this either means that one set is wrong, or that there was more than one variant.  The multiple variants hypothesis seems more plausible to me for 3 reasons:
* Both sets of numbers correspond to about the same fuel flow, ~51 kg/s (i.e. turbomachinery is the same with a different nozzle)
* Space Launch Report refers to the engine as 405H-2, which suggests a variant
* Pictures I can find do seem to show two nozzle variants: [#1](http://www.alternatewars.com/BBOW/Space_Engines/GE_Vega_Engine.png) and [#2](http://www.b14643.de/Spacerockets_2/United_States_4/Atlas-Vega/Gallery/Vega_1.jpg)

I think it makes sense to up the base ignitions from 2 to 3 given that (3) states specifically that this was the number of ignitions.

But would it be reasonable to have both variants in RO?  It seems like any changes happened before the planned first flight in 1960, but it does seem like both actually existed at some point in time.

If so, would it make sense to change any stats besides the thrust and Isp?  The static test suggests that the engine could have burned longer and with more ignitions but (a) The rated stats probably wouldn't have affected that (b) Ignitions would have been limited by onboard ignition fluid and (c) We don't really currently have a way to specify what the overburn penalty is (it's hard-coded currently).

And if we did have two variants, what nodes would they unlock in?  Source (1) suggests that the Atlas-Vega program both started and ended in 1959

**Comment by raidernick** [comment]
fixed here: https://github.com/KSP-RO/RealismOverhaul/pull/1836

**Comment by blowfishpro** [comment]
These were general comments about the X-405H's stats, configuring the BDB one is orthogonal to that

**Comment by NathanKell** [comment]
I would do it as two configs, with the H-2 rated to burn longer.

**Comment by blowfishpro** [comment]
Additional sources:

[Atlas-Vega: An Early NASA Launch Vehicle Development](https://cdn.discordapp.com/attachments/206181796164009984/539839828016234496/AtlasVegaJoelPowell.pdf)
[The Vega Program](https://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/19730064945.pdf)

**Comment by blowfishpro** [comment]
@pianojosh managed to get [a fragment of Convair's final Vega report](https://cdn.discordapp.com/attachments/541475381157298177/555150282405969920/Atlas_Vega_AP60-0294_partial_001_1.pdf), I would consider this to be more accurate than many of the other sources.  The stats have been updated to match it in #2011 

Still less sure about more configs though.  I did some measurements on photos and schematics and it doesn't look like the 2 segment and 4 segment nozzle and they don't really appear to be measurably different sizes, so that explanation is out for the Isp difference between sources.  It could be something internal, like a better injector, but I'll wait to get more info before doing anything like this.
